source/renderer/app/components/wallet/WalletReceive.scss

```diff
@@ -164,6 +164,13 @@
   h2 {
     font-family: var(--font-medium);
 
+    .hideUsed {
+      width: 118px;
```

Comment:
@daniloprates I'm not sure if the usage of hardcoded width is a good idea - especially if you think about all the possible translations. I would rather use `flexbox` with some width percentage and not `float`.

Reply:
Yes that is the ugly bit. I will think about it and let you know.